pkg/device/device-util.go

```diff
@@ -32,6 +32,9 @@ import (
 	apis "github.com/openebs/device-localpv/pkg/apis/openebs.io/device/v1alpha1"
 )
 
+// Compiled Regex to Ignore the Block devices
+var IgnoreBlockDevicesRegex *regexp.Regexp = nil
```

Comment:
Instead of using and setting a variable directly, can we have a config struct in this package, which contains all the config for the device driver.  I believe that will be a much cleaner approach in the long run.

Reply:
done